packaging/linux/test/README.md

```diff
@@ -53,6 +56,35 @@ After that you can upgrade it:
     cd /root/build/deb/amd64
     dpkg -i `ls -tr *.deb | tail -1`
 
+##### Local repo
```

Comment:
are these notes for the packager or for the end user?

Reply:
Notes for packager, they're just for testing upgrade paths without code signing key.